Content.Server/GameObjects/EntitySystems/MoverSystem.cs

```diff
@@ -174,8 +175,9 @@ private void UpdateKinematics(ITransformComponent transform, IMoverComponent mov
                     transform.LocalRotation = mover.VelocityDir.GetDir().ToAngle();
                     return;
                 }
-                (physics.Controller as MoverController)?.Move(mover.VelocityDir,
-                    mover.Sprinting ? mover.CurrentSprintSpeed : mover.CurrentWalkSpeed);
+
+                var speed = mover.Sprinting ? mover.CurrentSprintSpeed : mover.CurrentWalkSpeed;
+                (physics.Controller as MoverController)?.Move(mover.VelocityDir, speed);
```

Comment:
Nit, but the previous version made it clear that its only going to be used once, and the name doesn't add much as both branches of the ternary expression contain "Speed" anyways.